### PR TITLE
Added CRONTRIBUTING file

### PR DESCRIPTION
**Pull request by ninadbstack**
Added contributing file so that open source contributors have some guidelines while creating PRs

### CODE REVIEW

**Review by pankaj443 — APPROVED** [review]
LGTM

**Review by prklm10 — APPROVED** [review]

File: CRONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
+# Contributing to @percy/agent
```
Comment (prklm10) [inline CRONTRIBUTING.md:1]: Need to change the name to @percy/storybook

File: CRONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+- **100%** test coverage
+- [Valid commit message(s)](#commit-message-guidelines)
+
+### Commit message guidelines
+
+#### Atomic commits
+
+If possible, make [atomic commits](https://en.wikipedia.org/wiki/Atomic_commit), which means:
+- a commit should contain exactly one self-contained functional change
+- a functional change should be contained in exactly one commit
```
Comment (prklm10) [inline CRONTRIBUTING.md:62]: Both line means the same thing right?

Comment (ninadbstack) [reply CRONTRIBUTING.md:62]: Yes, thats how atomic commits are described in wikipedia article. A -> B and B -> A

**Pull request merged**